pom.xml

```diff
@@ -487,12 +487,6 @@
         <version>${jetty.version}</version>
         <scope>provided</scope>
       </dependency>
-      <dependency>
```

Comment:
Since this dependency is no longer provided in Jetty 10, will deleting it have a negative impact on Apache Spark? Would we need to find alternative ways to replace the function of `jetty-continuation`?

Reply:
Nope, I don't think since the Spark is not calling the module directly. I don't see any negative impact on this.
For this I have asked on the Jetty project also, see the last reply from:
https://github.com/jetty/jetty.project/issues/4934
In the future, if we need any of the `continuation` functionality, we should call the method from Servlet API instead.